lib/ui/checkbox.tsx

```diff
@@ -7,22 +7,33 @@ const Checkbox = React.forwardRef<
   React.ElementRef<typeof CheckboxPrimitive.Root>,
   React.ComponentPropsWithoutRef<typeof CheckboxPrimitive.Root>
 >(({ className, ...props }, ref) => (
-  <CheckboxPrimitive.Root
-    ref={ref}
-    className={cn(
-      "border-primary ring-offset-background focus-visible:ring-ring data-[state=checked]:bg-primary peer h-4 w-4 shrink-0 rounded-2xs border disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:text-f1-foreground",
-      focusRing(),
-      className
-    )}
-    {...props}
-  >
-    <CheckboxPrimitive.Indicator
-      className={cn("flex items-center justify-center text-current")}
+  <div className="flex items-center gap-4">
+    <CheckboxPrimitive.Root
+      ref={ref}
+      id="checkbox"
```

Comment:
the id must be unique, otherwise we'll face id collision when we have more than one checkbox on the screen.

Since we went the `htmlFor` way for the label, I think we should require `name` or `id` property to always have a unique name coming from the use case.